src/app/project-overview/create-resource-panel/component.ts

```diff
@@ -84,6 +87,10 @@ export class CreateResourcePanelComponent implements OnInit, OnDestroy {
     return MemberUtils.hasPermission(this._user, this._currentGroupConfig, View.Clusters, Permission.Create);
   }
 
+  get canCreateClusterTemplate(): boolean {
```

Comment:
I think we don't need this since "Create Cluster Template" opens the Cluster wizard so `canCreateCluster` is fine.

Reply:
Okay, btw.. on the cluster template page, it is being checked for the button 
<img width="334" alt="image" src="https://user-images.githubusercontent.com/56511988/186422653-59dcecfa-d9ed-4d5d-9830-d3c6eb4c0150.png">